### PR TITLE
(SLV-441) Configure nginx to serve gatling repos

### PR DESCRIPTION
**Pull request by RandellP**
Set autoindex on so we can navigator directories
created /usr/share/nginx/html/users/ to hold links to users gatling directories.
Setup nginx to open to that dir by default

### CODE REVIEW

**Comment by RandellP** [comment]
Small change, only need one review.  Whoever gets there first. :)

File: site-modules/gplt_runner_setup/manifests/core.pp [hunk 1]
```diff
@@ -46,7 +46,22 @@
...
 
-  include nginx
+  class { "nginx":
+    manage_repo    => true,
+    package_source => "nginx-stable",
+    confd_only     => true,
+  }
+
+  nginx::resource::server{ "default":
+    www_root => "/usr/share/nginx/html/users",
```
Comment (johnduarte) [inline site-modules/gplt_runner_setup/manifests/core.pp]: Should this resource just be moved to the user setup, set to  `"/home/${test_user}/gatling"`, and do away with link wrangling?

File: site-modules/gplt_runner_setup/manifests/core.pp [hunk 2]
```diff
@@ -46,7 +46,22 @@
...
+    autoindex => "on",
+  }
+
+  file { "htlm users":
```
Comment (johnduarte) [inline site-modules/gplt_runner_setup/manifests/core.pp]: s/htlm/html/

File: site-modules/gplt_runner_setup/manifests/user_setup.pp [hunk 3]
```diff
@@ -51,6 +57,12 @@
     group  => $test_user,
   }
 
+  file { "${test_user} html users link":
+    ensure => "link",
```
Comment (johnduarte) [inline site-modules/gplt_runner_setup/manifests/user_setup.pp]: I have had issues with nginx being able to traverse linked directories.  Has this been tested?  Also, should we just make this the default `www_root` directory in nginx and not not link it?

File: site-modules/gplt_runner_setup/manifests/core.pp (hunk 1 above)
Comment (RandellP) [reply site-modules/gplt_runner_setup/manifests/core.pp]: So the idea is there could be multiple users setup on the instance.  This creates the base users dir to hold the links.

File: site-modules/gplt_runner_setup/manifests/core.pp (hunk 2 above)
Comment (RandellP) [reply site-modules/gplt_runner_setup/manifests/core.pp]: fixed

File: site-modules/gplt_runner_setup/manifests/user_setup.pp (hunk 3 above)
Comment (RandellP) [reply site-modules/gplt_runner_setup/manifests/user_setup.pp]: I have as well, took some work to figure it out.  selinux was the main thing breaking it.  Which is why selinux is turned down as part of the instance setup.  I did test this on a new setup and it worked as expected.   And as for making it the default... see the comment above about multiple users.

**Pull request merged**